api/libproject/devstart.rb

```diff
@@ -1124,6 +1124,46 @@ def fetch_firecloud_user_profile(cmd_name, *args)
   :fn => ->(*args) {fetch_firecloud_user_profile("fetch-firecloud-user-profile", *args)}
 })
 
+def fetch_workspace_details(cmd_name, *args)
+  common = Common.new
+  ensure_docker cmd_name, args
+
+  op = WbOptionsParser.new(cmd_name, args)
+  op.opts.project = TEST_PROJECT
+
+  op.add_typed_option(
+      "--projectId=[projectId]",
+      String,
+      ->(opts, v) { opts.projectId = v},
+      "The Workspace to fetch details for (e.g. 'aou-rw-231823128'")
```

Comment:
I'd avoid putting real identifiers into this public repo if we can. Also, it'd be good to be explicit about whether this is a FC workspace name.

Reply:
its fake! I made it up but I wanted it to be similar enough to what a real workspace looks like so the example has some value.

Reply:
Project ID, not Workspace